src/react-apps/applications/service-development/src/features/appPublish/containers/releaseContainer.tsx

```diff
@@ -1,24 +1,528 @@
+    height: '100%',
+  },
+  cannotCreateReleaseTitle: {
+    padding: '1.2rem',
+  },
+  cannotCreateReleaseSubTitle: {
+    padding: '1.2rem',
+    fontSize: '1.4rem',
+  },
+  renderCannotCreateReleaseIcon: {
+    paddingTop: '2rem',
+  },
+});
+
+export interface IAppReleaseContainer extends WithStyles<typeof styles> { }
+
+function AppReleaseContainer(props: IAppReleaseContainer) {
+  const { classes } = props;
+  const [tabIndex, setTabIndex] = React.useState(0);
+  const [anchorElement, setAchorElement] = React.useState<Element>();
+
+  const [popoverOpenClick, setPopoverOpenClick] = React.useState<boolean>(false);
+  const [popoverOpenHover, setPopoverOpenHover] = React.useState<boolean>(false);
 
-export default function() {
   const appReleases: IAppReleaseState = useSelector((state: IServiceDevelopmentState) => state.appReleases);
+  const repoStatus: IRepoStatusState = useSelector((state: IServiceDevelopmentState) => state.repoStatus);
+  const handleMergeConflict: IHandleMergeConflictState =
+    useSelector((state: IServiceDevelopmentState) => state.handleMergeConflict);
+  const language: any = useSelector((state: IServiceDevelopmentState) => state.language);
 
   React.useEffect(() => {
-    AppReleaseActions.getAppReleases();
+    const { org, app } = window as Window as IAltinnWindow;
+    AppReleaseActions.getAppReleasesIntervalStart();
+    if (!language) {
+      FetchLanguageActionDispatchers.fetchLanguage(languageUrl, 'nb');
+    }
+    RepoStatusActionDispatchers.getMasterRepoStatus(org, app);
+    HandleMergeConflictActionDispatchers.fetchRepoStatus(getRepoStatusUrl(), org, app);
+    return () => {
+      AppReleaseActions.getAppReleasesIntervalStop();
+    };
   }, []);
 
-  if (!appReleases.releases || !appReleases.releases.length) {
+  function handleChangeTabIndex(event: React.ChangeEvent<{}>, value: number) {
+    setTabIndex(value);
+  }
+
+  function handlePopoverKeyPress(event: React.KeyboardEvent) {
+    if (event.key === 'Enter' || event.key === ' ') {
+      if (!anchorElement) {
+        setAchorElement(event.currentTarget);
+      }
+      setPopoverOpenClick(!popoverOpenClick);
+    }
+  }
+
+  function handlePopoverOpenClicked(event: React.MouseEvent) {
+    if (!anchorElement) {
+      setAchorElement(event.currentTarget);
+    }
+    setPopoverOpenClick(!popoverOpenClick);
+  }
+
+  function handlePopoverOpenHover(event: React.MouseEvent) {
+    setAchorElement(event.currentTarget);
+    setPopoverOpenHover(true);
+  }
+
+  function handlePopoverClose() {
+    if (popoverOpenHover) {
+      setPopoverOpenHover(!popoverOpenHover);
+    }
+  }
+
+  function renderCannotCreateRelease() {
+    return (
+      <Grid
+        container={true}
+        direction={'column'}
+        className={classes.cannotCreateReleaseContainer}
+        spacing={1}
+      >
+        <Hidden
+          mdDown={true}
+        >
+          <Grid
+            item={true}
+            xs={1}
+          >
+            <AltinnIcon
```

Comment:
No shared component for this error component?

Reply:
Not at this point.